Blog: Why we made chat our default UI (#16426)
PostHog/posthog.com: contents/blog/why-chat.md

---
title: Why we made chat our default UI
date: 2026-04-22
author:
  - ian-vanagas
featuredImageType: full
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2026_04_17_at_11_06_36_2x_7007f77b71.png
tags:
  - Inside PostHog
---

"What can I help you with today?"

> It's what any business would ask a customer walking into their shop, so why is it so controversial when a piece of software does it?

Reviewer: ```suggestion
It's what most businesses would ask when a customer walks into their shop, so why is it so controversial when a piece of software does it?
```

> It's what most businesses would ask when a customer walks into their shop, so why is it so controversial when a piece of software does it?

Reviewer: i think this sentence is still making the intro sound defensive

> Companies like Linear, Attio, and us (PostHog) have all made an [AI chat](/ai) the default homepage experience recently and some are not happy about it.

Reviewer: ```suggestion
Companies like Linear, Attio, and PostHog have all made [AI chat](/ai) the default homepage experience recently – and developers have opinions.
```
slight wording changes to make less defensive

![Complaints](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/Clean_Shot_2026_04_17_at_11_14_10_2x_b8266e901c.png)

> We're seeing good results from doing this so far. Only 6% of homepage visitors clicked a dashboard link while 12% chat with AI. On top of this, only 2% of people over 30 days have put the default dashboard back within one hour of visiting.

Reviewer: ```suggestion
Not everyone is happy about it, but our data shows promising results so far. Only 6% of homepage visitors clicked a dashboard link while 12% chat with AI. On top of this, only 2% of people over 30 days have put the default dashboard back within one hour of visiting.
```

To go further, we're explaining why we made this change and what we're doing to support it.

## Our existing default doesn't make sense anymore

PostHog started as an "open source product analytics tool" but has grown to be much more. 48.5% of active PostHog users in the last 30 days didn't even use product analytics. They rely on [session replay](/session-replay), [web analytics](/web-analytics), [feature flags](/feature-flags), [error tracking](/error-tracking), or any of our other tools. 

> This is great as it shows we're making progress towards our mission of providing every tool engineers need to build successful products. We're sure aiming for it. We had 48 [changelog-worthy](/changelog) updates in March and LLMs are increasing the velocity at which we can ship new features.

Reviewer: ```suggestion
This is great as it shows we're making progress towards our mission of providing every tool engineers need to build successful products. We sure are aiming for it. We had 48 [changelog-worthy](/changelog) updates in March, and LLMs are increasing the velocity at which we can ship new features.
```

But this shift and growth causes three problems:

1. Defaulting to a product analytics dashboard doesn't make sense for ~50% of our users.

> 2. There's a limited amount of room on the screen to put all of these products and features. 

Reviewer: ```suggestion
2. There isn't enough room on the screen to put all of these products and features. 
```

Reviewer: eliminates repetitive

3. Users have a limited amount of time and attention to learn the usage and best practices of these products too. 

> The combination of these creates the complaint that PostHog is "too complicated" and our UI is "too cluttered." Our [Platform UX team](/teams/platform-ux) is working on this, and we also have an exec leading a project on fixing "papercuts," 51 of which have been fixed in the last 2 weeks.

Reviewer: ```suggestion
As a result, a common complaint we've been hearing is that PostHog is "too complicated" and our UI is "too cluttered." Our [Platform UX team](/teams/platform-ux) is working on this and we also have an exec leading a project on fixing "papercuts," 51 of which have been fixed in the last 2 weeks.
```

1. A chat-based agent provide a much simpler UI. Users can use natural language to get the insights as well as create and modify flags, experiments, surveys, and more.
2. Agents can be set up with the [right tools and skills](/newsletter/agent-first-product-engineering) to figure out what products to use and know the best practices of using them.
3. Because they can use all of our products and features, a chat-based agent is more relevant to more of our users than a product analytics dashboard is.

The key bit is *when they work well*. Many people have had bad experiences with these chat-based agents, which is where resentment comes from. We've done a lot of work and testing and had enough positive feedback to be confident in ours though.

Making chat the default also helps us make improvements faster. More usage means more feedback, and the better tools and skills we build thanks to this feedback then spill over to our other agentic surfaces (like [MCP](/docs/model-context-protocol)).

## Chat has it's own challenges but is a step towards the future

> Chat has it's own challenges.

Reviewer: ```suggestion
Chat has its own challenges.
```
